docs/bootstrap.md

```diff
@@ -1,35 +1,56 @@
-# Bootstrapping a Kubernetes cluster
+# Bootstrapping Dragonfly on Digital Ocean
 
-## Create namespaces
+## Prerequisites
 
-```bash
-kubectl apply -f kubernetes\manifests\cert-manager\namespace.yaml
-kubectl apply -f kubernetes\manifests\discord\namespace.yaml
-kubectl apply -f kubernetes\manifests\dragonfly\namespace.yaml
+You will need:
+
+* [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/) configured with a...
+* [Digital Ocean Personal Access Token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) with at least the following scopes:
+  * kubernetes (2): create, delete
+  * If replacing an existing cluster:
+    * load\_balancer (1): delete
+* PWD set to the root of this repo
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [`helm`](https://helm.sh/docs/intro/install/)
+
+## Create the cluster in Digital Ocean
+```
+doctl k8s cluster create \
+  <cluster-name> \
+  --size s-2vcpu-2gb \
+  --count 2 \
+  --region nyc3
 ```
 
-## Install the Helm Chart to get all the dependencies
+Adjust `size` and `count` as needed. See [`doctl kubernetes cluster create
+docs`](https://docs.digitalocean.com/reference/doctl/reference/kubernetes/cluster/create/)
+for more options.
+
+## Apply `cert-manager` CRDs
 
-```bash
-helm install -f kubernetes\chart\production.yaml vipyrsec kubernetes\chart\
+```
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.crds.yaml
 ```
 
-## Apply the Discord bot deployment
+## Install the Helm Chart for `ingress-nginx` and `cert-manager`
 
 ```bash
-kubectl apply -f kubernetes\manifests\discord\bot
+helm upgrade vipyrsec ./kubernetes/chart --install
 ```
 
-## Apply the Dragonfly Mainframe deployment
+## Apply manifests and secrets
+
+Secrets are not included in this repo; you will need to create your own.
 
-```bash
-kubectl apply -f kubernetes\manifests\dragonfly\client
+```
+kubectl apply -f ./kubernetes/manifests -R
 ```
 
-After the mainframe ingress is created, you will need create the DNS records before deploying the client.
+## If access over the internet is required, create a DNS A Record to use the new load balancer
 
-## Apply the Dragonfly client deployment
+## If replacing an existing cluster, destroy old resources
 
-```bash
-kubectl apply -f kubernetes\manifests\dragonfly\mainframe
+```
+doctl k8s cluster delete <name>
+doctl compute load-balancer delete <id>

```

Comment:
`suggestion(non-blocking)`: Getting the ID is an unnecessary step.

```suggestion
doctl compute load-balancer delete <name>
```

Reply:
Hm, this docs page says you need `<id>`, https://docs.digitalocean.com/reference/doctl/reference/compute/load-balancer/delete/.

Reply:
Huh, weird. Where do you get the ID from? The UI?